include/xwayland/xwm.h

```diff
@@ -5,6 +5,13 @@
 #include <wlr/xwayland.h>
 #include <xcb/render.h>
 
+#ifdef WLR_HAS_XCB_ICCCM
```

Comment:
Can you change `#ifndef WLR_XWM_H` to be `#ifndef XWAYLAND_XWM_H`?

Reply:
Also, `wlr_xwm_atoms_contains` should probably be renamed to `xwm_atoms_contains` for consistency (if it is present in the symbols file, it should be removed from there too).

I don't remember why we put `wlr_xwm` in the public headers, it seems to make sense to keep it private. @acrisci comments?

Reply:
Done and done. This was one of the "auto symbol" so will remove itself with name change on rebuild.
I believe the header just got moved together with `xwayland.h` because I had it incorrectly in the source tree at start and @acrisci moved both away, please say if you see a reason for it to be public, I can do some more cleanup.

Reply:
xwm_atoms_contains was intended to be somewhat public.
It's currently a helper exclusivly used for the `is_unmanaged` function, but it could also be used to e.g. detect dzen2 (which will have `NET_WM_WINDOW_TYPE` as `NET_WM_WINDOW_TYPE_DOCK` but I think no more overrdie redirect if called with `-dock` argument). 

It's declared in  `include/wlr/xwm.h` this should change if it's non-public.
Since the headers under `include/wlr` are installed with the normal install, all functions declared there should be public, otherwise it's just confusing.

Reply:
Yes, I moved xwm.h to `include/xwayland/xwm.h` which is private - if this function was meant to be public I'll move it to `xwayland.h` instead of renaming, will amend the last commit.

Reply:
Sounds good. A public function should probably return the `xcb_atom_t` in question, so the value can be read, instead of just being bool.
But this can be done, when we have a usecase.

Reply:
Ok, let's keep this as it is now and either of us can make it public properly later, it's a bit of a bigger change than just moving a line.

Reply:
File a ticket for follow-up please